release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -88,6 +88,12 @@ This release introduces the following updates to the *Developer* perspective of
 [id="ocp-4-16-networking"]
 === Networking
 
+[id="ocp-4-16-networking-multiple-cidr"]
+==== Define multiple CIDR blocks for one network security group rule
+
+With this release, you can now create multiple CIDR blocks with one network security group (NSG) for Ingress controllers in {product-title} clusters hosted on Azure. You can do this using the `allowedSourceRanges` field. Creating multiple Ingress controllers with the `allowedSourceRanges` field configured will allow those Ingress controllers to share the maximum limit of 4000 rules between them.
```

Comment:
@gcs278 can you PTAL at this RN entry? would appreciate your feedback, thank you 😄